docs/editors-guide/exclusion-reasons.md

```diff
@@ -1,33 +1,35 @@
 # Exclusion Reasons
 
 ## Description 
-_This document is a work in progress._ Noted below are reasons for excluding external ontology or terminology terms from Mondo. See enum file [here](https://github.com/monarch-initiative/mondo/blob/master/src/schema/mondo.yaml).
+_This document is a work in progress._ Noted below are reasons for excluding external ontology or terminology terms from Mondo. See enum file [here](https://github.com/monarch-initiative/mondo/blob/master/src/schema/mondo.yaml). The dashes in column 1 are meant to indicate the code is a subtype of the code above.
```

Comment:
@sabrinatoro I added this additional line

Reply:
great point! thank you!